### DISCUSSION TITLE
I need to change my school to the sub location

### DISCUSSION THREAD

**Discussion by micah4232** [community/community#55806] [GitHub Education]
### Hi everyone,

Hello community,

I have an issue that when I signed up for github it let me pick the Hub for our district SCCCD and I dint pick the Madera Community College branch (either it didnt come up when i searched or it forced me to choose SCCCD based on the geo location it was using or something) and now that i want to verify to get to test the github copilot out it says my student ID and documents aren't sufficient bc it doesnt match the same school i chose. Is there anyway to fix this? would love and appreciate all the help you could offer TIA. Have great day. Please and thank you.

Regards.

**Comment by samus-aran** [comment]
Hi @micah4232 

You can cancel your current submission and re-apply selecting the correct school. Yes, the system does detect associated domains to institutions. If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.
